ewc-rfc001-issue-verifiable-credential.md

```diff
@@ -412,12 +447,6 @@ Query params for the authorisation request are given below:
    <td>The value must be ‘code’
    </td>
   </tr>
-  <tr>
-   <td><code>scope</code>
-   </td>
-   <td>The value must be ‘openid’
-   </td>
-  </tr>
   <tr>
    <td><code>state</code>
```

Comment:
The `state´ parameter is not specified in draft 13 nor in HAIP, but it is REQUIRED in the EBSI specification. We have to decide how to handle this.

Reply:
Same comment as before

Reply:
Will remove this.